preupg/ui/report/service.py

```diff
@@ -44,16 +44,26 @@ def remove_upload(path):
 
 def extract_tarball(tbpath, target_dir):
     """ return report paths: (xml, html) """
-    tar = tarfile.open(tbpath)
+    def strip_subfolder(tar_content):
+        """The Web UI does not expect the subfolder in the extracted tarball
+        content.
+        """
+        for member in tar_content:
+            path_wo_subfolder = member.path.split(os.sep)[1:]
```

Comment:
Seems like typo `path_to_subfolder`.

Reply:
It's meant is without :) I'll change it.